src/rastervision/common/data/generators.py

```diff
@@ -1,8 +1,9 @@
 import numpy as np
+import codecs, json
+from os.path import exists, join
 
-from rastervision.common.utils import get_channel_stats
 from rastervision.common.settings import TRAIN, VALIDATION, TEST
-
+from rastervision.common.utils import get_channel_stats, save_json
```

Comment:
There's a standard order for grouping imports. First are packages that are built in to Python. Second are third party packages. Third are packages within your project. I've also been grouping our own packages by common, semseg or tagging, just to make it a little easier to read.